routes/api/v1/favorites.js

```diff
@@ -27,4 +27,22 @@ router.post('/', (request, response) => {
     })
 });
 
+router.get('/:id', (request, response) => {
+  let songId = request.params.id
+  database('favorites').where('id', songId).first()
+    .then(songRecord => {
+      if (songRecord) {
+        database('favorites')
```

Comment:
This might be a good place to refactor later to split into a helper method

Reply:
Agreed!